marketplace_builder/views/pages/get-started/partner-portal/setting-up-production-domain.liquid

```diff
@@ -0,0 +1,82 @@
+---
+converter: markdown
+metadata:
+  title: Setting Up a Production Domain
+  description: This guide will help you set up a production domain. 
+  slug: get-started/partner-portal/setting-up-production-domain
```

Comment:
not sure - but there should be no indentation for slug - it makes page inaccessible 
@diana-lakatos

Reply:
Yep

Reply:
@godot @pavelloz Thank you! Fixed.